src/dbally/views/freeform/text2sql/_autodiscovery.py

```diff
@@ -144,6 +144,17 @@ def generate_description_by_llm(self, example_rows_cnt: int = 5) -> Self:
         self._description_extraction = _LLMSummaryDescriptionExtraction(example_rows_cnt)
         return self
 
+    def suggest_similarity_indexes(self) -> Self:
```

Comment:
Why is such flag-setter method preferable to this being just a construction argument?

Reply:
Just to stick to the fluent api pattern that is used while configuring autodiscovery, also it may be possible in the future to have some extra configuration on this feature